documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,99 @@
+![Test environment](assets/knotx-test-environment.png)
+
+### System under test
+* 1 x Apache - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Knot.x - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
+
+### Knotx author start command
+`java -Dlogback.configurationFile=/content/knotx/author/logback.xml -Dvertx.cacheDirBase=/content/knotx/author/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18092 -Dcom.sun.management.jmxremote.rmi.port=18092 -Djava.rmi.server.hostname=10.254.165.236 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/author/knotx.jar -conf /content/knotx/author/config/author-config.json`
+
+### Knotx publish start command
+`java -Dlogback.configurationFile=/content/knotx/publish/logback.xml -Dvertx.cacheDirBase=/content/knotx/publish/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18093 -Dcom.sun.management.jmxremote.rmi.port=18093 -Djava.rmi.server.hostname=10.254.165.236 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/publish/knotx.jar -conf /content/knotx/publish/config/publish-config.json`
+
+### Knotx mocks start command
+`java -Dlogback.configurationFile=/content/knotx/mocks/logback.xml -Dvertx.cacheDirBase=/content/knotx/mocks/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=19000 -Dcom.sun.management.jmxremote.rmi.port=19000 -Djava.rmi.server.hostname=10.254.165.163 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/mocks/knotx.jar -conf /content/knotx/mocks/config/mocks-config.json`
+
+### Test harness
+* 1 x Jmeter Master - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+* 2 x Jmeter Servers - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+
+### Repositories
+* mocks-knotx - \knotx-performance-tests\config-knotx\
```

Comment:
Make full link to the folder on github, e.g. using markdown as below
```
* mocks-knot - [knotx-perfromance-tests\config-knotx](https://github.com/Cognifide/knotx/tree/master/knotx-perfromance-tests/config-knotx)
```

Same for next bullet